sync/etcd/etcd.go

```diff
@@ -163,6 +164,63 @@ func eventsFromNode(action string, node *etcd.Node, responseChan chan *sync.Even
 }
 
 //Watch keep watch update under the path
+func (s *Sync) WatchEx(path string, responseChan chan *sync.Event, stopChan chan bool, revision int64) error {
+	var etcdResponseChan chan *etcd.Response
+
+	etcdResponseChan = make(chan *etcd.Response)
```

Comment:
etcdResponseChan := make(chan *etcd.Response) will be enough (no need to declare it separately)